### PR TITLE
Mark SE-0292 and SE-0321 as implemented in Swift 5.7

### PR DESCRIPTION
**Pull request by rjmccall**
Per personal communication with Tom Doron

### CODE REVIEW

**Comment by rjmccall** [comment]
Seeking review from the proposal authors: @clarkbw, @whitneyimura, and @mattt.  I apparently cannot `@` Bryan and Whitney or invite them to review the proposal, though.

**Review by mattt — APPROVED** [review]

File: proposals/0292-package-registry-service.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
            [Whitney Imura](https://github.com/whitneyimura),
            [Mattt Zmuda](https://github.com/mattt)
 * Review Manager: [Tom Doron](https://github.com/tomerd)
-* Status: **Accepted (2021-06-23)**
+* Status: **Implemented (Swift 5.7)**
```
Comment (tomerd) [inline proposals/0292-package-registry-service.md:8]: The SwiftPM side of this was implemented in 5.7

File: proposals/0321-package-registry-publish.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 * Authors: [Whitney Imura](https://github.com/whitneyimura),
            [Mattt Zmuda](https://github.com/mattt)
 * Review Manager: [Tom Doron](https://github.com/tomerd)
-* Status: **Accepted (2021-09-01)**
+* Status: **Implemented (Swift 5.7)**
```
Comment (tomerd) [inline proposals/0321-package-registry-publish.md:7]: The SwiftPM / client side of this was _not_ implemented yet.

**Comment by rjmccall** [comment]
Okay.  I made a new PR to only change SE-0292; closing this one.

**Pull request closed without merging**